random-ballerina/random_errors.bal

```diff
@@ -16,3 +16,6 @@
 
 # Represents Random module related errors.
 public type Error distinct error;
+
+# Represents the arithmetic error.
+public type ArithmeticError distinct Error;
```

Comment:
Do we use this anywhere

Reply:
Yes, We are creating an arithmetic error on the native side but Ballerina has only a distinct error as `Error`. So Ballerina can't catch that error.

- https://github.com/ballerina-platform/module-ballerina-random/blob/main/random-native/src/main/java/org/ballerinalang/stdlib/random/nativeimpl/ExternMethods.java#L49
- https://github.com/ballerina-platform/module-ballerina-random/blob/main/random-native/src/main/java/org/ballerinalang/stdlib/random/nativeimpl/Constant.java#L28

Reply:
Ack.